dart/lib/src/sentry_client.dart

```diff
@@ -99,6 +113,14 @@ class SentryClient {
       event = event.copyWith(exception: sentryException);
     }
 
+    if (_options.attachStackTrace &&
+        event.throwable == null &&
+        event.exception == null) {
+      final frames = _stackTraceFactory.getStackFrames(StackTrace.current);
+
+      event = event.copyWith(stackTrace: SentryStackTrace(frames: frames));
```

Comment:
frames can be null or empty, we dont want that

Reply:
✅